hasura/metadata/databases/default/tables/public_contributions.yaml

```diff
@@ -90,6 +90,26 @@ select_permissions:
               _is_null: true
       allow_aggregations: true
 event_triggers:
+  - name: contributionInsertCreateActivity
```

Comment:
might be nice to rename all these to have activity in the front so they are grouped in the UI

Reply:
ok, how about we name the trigger `activityHandler` since we probably will want to support more than just update in that same file.

and then we can rename these to `activityHandlerContributionInsert`

how's that sound?